stripe/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -141,39 +135,14 @@ internal abstract class BaseSheetViewModel<TransitionTargetType>(
         }
     }
 
-    fun transitionTo(target: TransitionTargetType) {
-        _userMessage.value = null
```

Comment:
This was moved to the `PaymentSheetActivity.onCreate` where the transition is observed.  If the button moves into the fragment this will no longer be needed